cypress/plugins/index.js

```diff
@@ -50,6 +51,8 @@ module.exports = (on, config) => {
 
   const waitUntilAppRestarts = async (timeout) => await waitOn({ delay: 2000, resources: [config.baseUrl], timeout })
 
+  const makeSureCypressCanInterpretTheResult = () => null
```

Comment:
As I commented here https://github.com/alphagov/govuk-prototype-kit/pull/1454/files#r915009116

I think it's worth passing through any non-undefined result unless you're sure it makes no difference.

Also, should this function be centralised in test helpers somewhere?

If I'm overcomplicating it I'm happy with it as-is and we can update if it causes a problem.